zerver/lib/test_classes.py

```diff
@@ -78,6 +80,20 @@
 if settings.ZILENCER_ENABLED:
     from zilencer.models import get_remote_server_by_uuid
 
+class _CapturingHandler(logging.Handler):
+    _LoggingWatcher = collections.namedtuple("_LoggingWatcher",["records", "output"])
```

Comment:
This should be inside `__init__` constructor, otherwise it will give `NameError: name '_LoggingWatcher' is not defined`.
Have you tested it before submitting pr.

Reply:
Please have a look at new solution posted.